nodeJS/mongoDB/intro_to_mongoDB.md

```diff
@@ -0,0 +1,24 @@
+Data persistence is integral to the vast majority of web applications. Local storage has limitations and poses security risks, and SaaS options such as Firebase work well, but ultimately being able to build, understand how to structure, and query your own database are important skills for any full stack developer to have. For example, consider Twitter. Where are your username and password are stored? Or perhaps where all of your Tweets might be fetched from? How does the application determine what Tweets are shown to you every time you log on? These operations are handled by a database. Databases are split into "relational" and "non-relational" types of databases, and each handles data and scaling in different manners. [This article](https://www.mongodb.com/scale/nosql-vs-relational-databases) goes in depth with the difference between the two. This lesson will teach you about MongoDB, a popular NoSQL database.
```

Comment:
-> "understand how to structure, build, and query your own database" would read a bit better better in my opinion.

"Where are your username and password __are__ stored" is grammatically incorrect.

Reply:
Whoops, thanks for the catch.